.travis.yml

```diff
@@ -95,14 +95,15 @@ before_install:
       source venv/bin/activate
       free -m
     elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      brew install gcc
+      brew install gcc ccache
+      export PATH="/usr/local/opt/ccache/libexec:$PATH"
       touch config.sh
       git clone https://github.com/matthew-brett/multibuild.git
       source multibuild/common_utils.sh
       source multibuild/travis_steps.sh
       before_install
-      export CXX=clang++
-      export CC=clang
+      export CXX="ccache clang++"
```

Comment:
Might be worth adding a comment here - I specified the compilers again because brew installing gcc was going to use gcc and g++.